### PR TITLE
[IE Python Speech Sample] Add `--scale_factor` and `--performance_counter` options

### PR DESCRIPTION
**Pull request by dpigasin**
### Details:
 - *There is the similar PR (#6650) that was opened earlier than mine, but it has some issues*

### Tickets:
 - *59516 (partly)*

### CODE REVIEW

**Review by generalova-kate — APPROVED** [review]

**Comment by generalova-kate** [comment]
Please sync with mentioned above PR and merge

**Review by tsavina — APPROVED** [review]

File: inference-engine/ie_bridges/python/sample/speech_sample/arg_parser.py [hunk 1]
```diff
@@ -28,10 +28,16 @@ def parse_args() -> argparse.Namespace:
...
+    args.add_argument('-sf', '--scale_factor', type=float,
+                      help='Optional. User-specified input scale factor for quantization.')
     args.add_argument('-wg', '--export_gna_model', type=str,
                       help='Optional. Write GNA model to file using path/filename provided.')
     args.add_argument('-we', '--export_embedded_gna_model', type=str, help=argparse.SUPPRESS)
     args.add_argument('-we_gen', '--embedded_gna_configuration', default='GNA1', type=str, help=argparse.SUPPRESS)
+    args.add_argument('-pc', '--performance_counter', action='store_true',
+                      help='Optional. Enables performance report (specify -a to ensure arch accurate results).')
+    args.add_argument('-a', '--arch', default='CORE', type=str.upper, choices=['CORE', 'ATOM'],
+                      help='Optional. Specify a architecture. CORE, ATOM with combination of -pc.')
```
Comment (generalova-kate) [inline inference-engine/ie_bridges/python/sample/speech_sample/arg_parser.py:40]:
```suggestion
                      help='Optional. Specify architecture. CORE, ATOM with the combination of -pc.')
```

File: inference-engine/ie_bridges/python/sample/speech_sample/arg_parser.py [hunk 2]
```diff
@@ -28,10 +28,16 @@ def parse_args() -> argparse.Namespace:
     args.add_argument('-bs', '--batch_size', default=1, type=int, help='Optional. Batch size 1-8 (default 1).')
     args.add_argument('-qb', '--quantization_bits', default=16, type=int,
                       help='Optional. Weight bits for quantization: 8 or 16 (default 16).')
+    args.add_argument('-sf', '--scale_factor', type=float,
+                      help='Optional. User-specified input scale factor for quantization.')
```
Comment (generalova-kate) [inline inference-engine/ie_bridges/python/sample/speech_sample/arg_parser.py:32]:
```suggestion
                      help='Optional. The user-specified input scale factor for quantization.')
```

File: inference-engine/ie_bridges/python/sample/speech_sample/README.md [hunk 3]
```diff
@@ -113,9 +115,18 @@ Options:
   -qb QUANTIZATION_BITS, --quantization_bits QUANTIZATION_BITS
                         Optional. Weight bits for quantization: 8 or 16
                         (default 16).
+  -sf SCALE_FACTOR, --scale_factor SCALE_FACTOR
+                        Optional. User-specified input scale factor for
```
Comment (generalova-kate) [inline inference-engine/ie_bridges/python/sample/speech_sample/README.md:119]:
```suggestion
                        Optional. The user-specified input scale factor for
```

File: inference-engine/ie_bridges/python/sample/speech_sample/README.md [hunk 4]
```diff
@@ -113,9 +115,18 @@ Options:
...
+                        quantization.
   -wg EXPORT_GNA_MODEL, --export_gna_model EXPORT_GNA_MODEL
                         Optional. Write GNA model to file using path/filename
                         provided.
+  -pc, --performance_counter
+                        Optional. Enables performance report (specify -a to
+                        ensure arch accurate results).
+  -a {CORE,ATOM}, --arch {CORE,ATOM}
+                        Optional. Specify a architecture. CORE, ATOM with
```
Comment (generalova-kate) [inline inference-engine/ie_bridges/python/sample/speech_sample/README.md:128]:
```suggestion
                        Optional. Specify architecture. CORE, ATOM with the
```

**Review by generalova-kate — APPROVED** [review]

**Pull request merged**